src/routes/blogRoute.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+import express from 'express';
+import path from 'path';
+import multer from 'multer';
+import {
+  blogPost, blogGet, blogDetail, blogDelete, BlogPatch, blogComment,
+}
+  from '../controllers/blogController';
+import auth from '../auth/checkAuth';
+
+const router = express.Router();
+
+const storage = multer.diskStorage({
+
+  destination(req, file, cb) {
+    cb(null, path.join(__dirname, '../uploads'));
+  },
+
+  filename(req, file, cb) {
+    cb(null, Date.now() + path.extname(file.originalname));
+  },
+});
```

Comment:
You need to update your app to use an online asset host like Cloudinary instead of saving all the images in your very limited database.